plugin.go

```diff
@@ -66,9 +72,24 @@ func (a Author) String() string {
 	return a.Username
 }
 
+func newCommitMessage(m string) Message {
+	// not checking the length here
+	// as split will always return at least one element
+	// check it if using more than the first element
+	splitMsg := strings.Split(m, "\n")
+
+	return Message{
+		msg:   m,
+		Title: strings.TrimSpace(splitMsg[0]),
+		Body:  strings.TrimSpace(strings.Join(splitMsg[1:], "\n")),
+	}
+}
+func (m Message) String() string {
+	return m.msg
```

Comment:
Can't this return just title + newline + body?

Reply:
there are some commit messages using an empty second line.

if we do in that way, we will lose the empty second line,
as the Message.msg is not high cost, so IMO, using it for saving the origin format is worthy.